### DISCUSSION TITLE
v3.5.7 breaks inline styles in the DOM

### DISCUSSION THREAD

**Discussion by jackwh** [livewire/livewire#8881] [7. Bugs (archived)]
### Livewire version

v3.5.7

### Laravel version

v11

### Which PHP version are you using?

PHP 8.3

### Steps To Reproduce

The `style` property is used to add height to Filament charts. Since v3.5.7, charts are loading with incorrect heights. Filament has since [locked Livewire](https://github.com/filamentphp/filament/issues/14315#issuecomment-2368391510) at v3.5.6.

This seems like an unintentional bug introduced in #8870. Further information is in the links below.

### Repository or Wirebox URL

https://github.com/livewire/livewire/pull/8870/files?diff=split&w=1#r1770798303

### Additional information

- https://github.com/filamentphp/filament/issues/14315
- https://github.com/livewire/livewire/pull/8870

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by whousehomestay** [comment]
Yeah, experiencing this too..

**Comment by jirkacisar** [comment]
It looks like all inline styles are ignored :-(

Editor:
![image](https://github.com/user-attachments/assets/0e4221c6-9792-47b9-b754-28e1ad646724)
Developer tools:
![image](https://github.com/user-attachments/assets/ff0d156e-b8c6-4ead-907f-119d2a847100)

**Comment by leandrocfe** [comment]
https://github.com/livewire/livewire/pull/8883

**Comment by ollosh** [comment]
Experiencing the same issues with the inline styles

**Comment by LucianoVandi** [comment]
Experiencing this too with [Power-Components/livewire-powergrid](https://github.com/Power-Components/livewire-powergrid), it's not possibile to hide/show columns dynamically.

**Comment by ghbob** [comment]
I also have problems were some properties don't seem to be reactive anymore:
A Blade `@if` condition isn't re-evaluated after a Livewire `render()`, oddly enough if I wrap it with a `<div>` it started working again 🤷 

However I also pinned to `3.5.6` for now.

@joshhanley (sorry about the ping!) 
The PR (https://github.com/livewire/livewire/pull/8883) apparently just rebuilds the js, any chance there was an issue with the build for 3.5.7? (https://github.com/livewire/livewire/commit/edf0b33424fb1066d70a93bbd4154cede3bf92c0#diff-71be7e35f51d605a8814367f54b7ac1ddf101170c889f2d2befe0d4cd7f60f9e)
 
another detailed issue: https://github.com/livewire/livewire/discussions/8904
related: https://github.com/livewire/livewire/discussions/8916

**Comment by ghbob** [comment]
Apparently fixed since the new build (3.5.10) removes these again:
https://github.com/livewire/livewire/commit/774092003edb2670615ef09f3a9fbdd335d6d0d7#diff-082bdedd3673dc44256e2be1be654a4ec9ee479e286fb8149186c278eb11b1eaL6823-L6824
https://github.com/livewire/livewire/commit/774092003edb2670615ef09f3a9fbdd335d6d0d7#diff-082bdedd3673dc44256e2be1be654a4ec9ee479e286fb8149186c278eb11b1eaL6832-L6833

These were added with the 3.5.7 https://github.com/livewire/livewire/pull/8870#pullrequestreview-23211708309.

But I couldn't find the change in Alpine itself: https://github.com/alpinejs/alpine/commits/main/packages/morph/src/morph.js


@calebporzio (sorry for the ping), any chance you (sometimes) build against your local version of Alpine which had this change?
Would explain why these references change from `node_modules/@alpinejs` to `../alpine/packages`:
https://github.com/livewire/livewire/commit/edf0b33424fb1066d70a93bbd4154cede3bf92c0#diff-082bdedd3673dc44256e2be1be654a4ec9ee479e286fb8149186c278eb11b1eaL6721-R6725


Thanks!
